quality-assurance/test-strategy.md

```diff
@@ -70,6 +70,13 @@ This section outlines the test types that need to be considered when preparing a
   * Pester tests to validate the environment is as expected.
   * Smoke and/or regression tests to ensure the product(s) are working in the environment as expected.
 
+### Disaster Recovery
+
+The disaster recovery plan should be tested for a new or significantly changed system.  This will normally consist of a drill, whereby the disaster scenario(s) are simulated, and the written disaster recovery plan is followed.  This is to verify:
```

Comment:
possible change "This will normally consist of a drill" to "This will normally consist of dry running through the steps"